angular/projects/catalyst/src/lib/formly/formly-checkbox/formly-checkbox.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component } from '@angular/core';
+import { FieldType, FieldTypeConfig } from '@ngx-formly/core';
+
+@Component({
+  selector: 'formly-checkbox',
+  template: `
+    <cat-checkbox
+      [formControl]="formControl"
+      [formlyAttributes]="field"
+      [checked]="props.checked"
+      [identifier]="props.identifier"
+      [indeterminate]="props.indeterminate"
+      [label]="props.label"
+      [name]="props.name"
+      [hint]="props.hint"
+      [labelLeft]="props.labelLeft"
+      [nativeAttributes]="props.nativeAttributes"
+      >
+    </cat-checkbox>
+  `,
+})
+export class CatCheckboxFieldType extends FieldType<FieldTypeConfig> {}
```

Comment:
please add newlines to each file ending. This is our standard and should lead to a lint error 🤔